src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -1,4 +1,55 @@
 package com.textmessenger.controller;
 
+import com.textmessenger.model.User;
+import com.textmessenger.service.UserService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/users")
 public class UserController {
+
+  private final UserService userService;
+
+  public UserController(UserService userService) {
+    this.userService = userService;
+  }
+
+  @PostMapping("/user")
```

Comment:
I don't think that creating CRUD methods while you don't need them is a good idea. I would prefer to see this controller class empty for now. First you need to design your backend REST API

Reply:
Its basic functionality. We are meeting at mondey with Kostya to design back end as well we can imagine.

Reply:
There's already a task for that. You shall discuss it with the whole team. For now you don't need any implementation, especially considering you don't have API design in place